src/apps/lwaftr/alarms/lwaftr-default-alarms.conf

```diff
@@ -0,0 +1,18 @@
+alarms {
+   alarm-inventory {
+       alarm-type {
+           alarm-type-id arp-resolution;
+           alarm-type-qualifier "";
+           has-clear true;
+           resource external-interface;
```

Comment:
Inconsistent has-clear / resource ordering here between the two entries..  Why is the qualifier listed here if it's an empty string?  Is the description something that this file should fill in?

Ideally the alarms that an app can issue should be associated with the app rather than the lwaftr.  WDYT?  Is there a way to do this?  Can be in a followup.